data/Dockerfiles/acme/docker-entrypoint.sh

```diff
@@ -192,7 +185,7 @@ while true; do
   done
 
   # Unique elements
-  ALL_VALIDATED=(${VALIDATED_MAILCOW_HOSTNAME} $(echo ${VALIDATED_CONFIG_DOMAINS[*]} ${ADDITIONAL_VALIDATED_SAN[*]} | xargs -n1 | sort -u | xargs))
+  ALL_VALIDATED=($(echo ${VALIDATED_MAILCOW_HOSTNAME} ${VALIDATED_SUBDOMAINS[*]} ${ADDITIONAL_VALIDATED_SAN[*]} | xargs -n1 | sort -u | xargs))
```

Comment:
The hostname should be first here, any reason you changed it?

Reply:
Yep, the reason is, your hostname might be mail.example.com and if you add mail.* you have to remove the duplicate. Otherwise ACME-verfication will not work.

Reply:
MAILCOW_HOSTNAME has to be the first one. That‘s what LE will use for the common name field, and we don‘t want one of the autoconfig domains to be used there. Please change this line back to what it used to be and instead delete MAILCOW_HOSTNAME from VAPIDATED_DOMAINS if it‘s in there.

Reply:
Yeah, that is a valid point, I haven't considered that